docs/src/pages.js

```diff
@@ -129,6 +129,11 @@ const pages = [
     ],
   },
   { title: 'Component API', ...findPages[0] },
+  {
+    title: 'Error Decoder',
+    pathname: '/error-decoder',
+    children: [{ pathname: '/error-decoder/[code]' }],
```

Comment:
> Do we need to generate a new page per error?

No. Though we have the option with `getStaticPaths`.

We put the `[code]` in the route because we know all the codes at build time and could pre-render the pages with a better skeleton.